release_notes/ocp-4-12-release-notes.adoc

```diff
@@ -2248,6 +2290,26 @@ $ ./openshift-install wait-for install-complete
 
 * When using the Technology Preview OCI feature with the oc-mirror CLI plug-in, the mirrored catalog embeds all of the Operator bundles, instead of filtering only on those specified in the image set configuration file. (link:https://issues.redhat.com/browse/OCPBUGS-5085[*OCPBUGS-5085*])
 
+* There is currently a known issue when you run the Agent-based {product-title} Installer to generate an ISO from a directory where the previous release was used for ISO generation. An error message is displayed with the release version not matching. As a workaround, create and use a new directory. (link:https://issues.redhat.com/browse/OCPBUGS-5159[*OCPBUGS#5159*])
+
+* The defined capabilities in the `install-config.yaml` file are not applied in the Agent-based {product-title} installation. Currently, there is no workaround. (link:https://issues.redhat.com/browse/OCPBUGS-5129[*OCPBUGS#5129*])
+
+* Pods in the `openshift-marketplace` cause PodSecurityViolation alerts in a vanilla {product-title} cluster. This is because the newly created project's `pod-security.kubernetes.io/warn` and audit are restricted, and the CatalogSource's `spec.grpcPodConfig.securityContextConfig` is not restricted by default, so you will get the related pod security admission (PSA) warnings, like below:
```

Comment:
[API object references](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#api-object-object-references) and ISG nit to avoid "vanilla":
```suggestion
* Pods in the `openshift-marketplace` namespace cause PodSecurityViolation alerts in a default {product-title} cluster. This is because the newly created project's `pod-security.kubernetes.io/warn` annotation and audit are restricted, and the CatalogSource's `spec.grpcPodConfig.securityContextConfig` field is not restricted by default, so you will get the related pod security admission (PSA) warnings, like below:
```
it's possible the object references above aren't right, I made an educated guess, so please feel free to change as needed!

Reply:
thanks I hadn't noticed this 😆 your suggestion looks right